app/views/books/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2> Books </h2>
+
+<%= @book.title %>
+
+<%= form_for [@book, @user] do |f| %>
+  <%= f.label :review, "Review" %>
+  <%= f.text_field :review %>
+
+  <%= f.submit %>
+<% end %>
+
+
+
+<% @book.reviews.reverse.each_with_index do |review, i| %>
```

Comment:
This is too much logic for a view. You'd want to make a method that lives on a Book instance that does this work.